msteams-platform/sbs-tabs-and-messaging-extensions-with-sso.yml

```diff
@@ -78,7 +78,7 @@ items:
     1. Enter required bot handle name in **Bot handle**.
     1. Select your **Subscription** from the dropdown list.
     1. From the **Resource group** dropdown list, select your existing resource group. You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
-
+       
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
space removed.